packages/launcher/src/renderer/styles.css

```diff
@@ -111,6 +112,19 @@ body,
   transform: perspective(400px) rotateY(15deg);
 }
 
+.app-editing {
+  transition: all 0.3s;
+  transform: perspective(400px) rotateY(-18deg);
+  animation: pulse 0.8s infinite both;
+}
+
+@keyframes pulse {
+
```

Comment:
White space is not consistent with the rest of the stylesheet, could you format it please?